grpc/src/main/java/com/linecorp/armeria/server/grpc/UnframedGrpcService.java

```diff
@@ -180,14 +180,14 @@ public HttpResponse serve(ServiceRequestContext ctx, HttpRequest req) throws Exc
                                RequestLogProperty.RESPONSE_CONTENT);
 
         final CompletableFuture<HttpResponse> responseFuture = new CompletableFuture<>();
-        req.aggregateWithPooledObjects(ctx.eventLoop(), ctx.alloc()).handle((clientRequest, t) -> {
+        req.aggregateWithPooledObjects(ctx.eventLoop(), ctx.alloc()).handleAsync((clientRequest, t) -> {
             if (t != null) {
                 responseFuture.completeExceptionally(t);
             } else {
                 frameAndServe(ctx, grpcHeaders.build(), clientRequest, responseFuture);
             }
             return null;
-        });
+        }, ctx.eventLoop());
```

Comment:
What happens if we revert this?

Reply:
I'm wondering about it too. 😄

Reply:
Explained at https://github.com/line/armeria/pull/3584#discussion_r640413691

Reply:
I thought this was different because we are specifying the `EventLoop` here. https://github.com/line/armeria/pull/3584/files#diff-bdfe29918f02ed24a4fd7e4ce1a0dc710ec18ee74f61d7d518533613d27f7b06R187 😄 
By the way, can we just do `ctx.push()` in order not to put the task into the queue?

Reply:
I had a similar idea. But, I followed `grpcResponse.aggregate().handleAsync(() -> ctx.eventLoop())` pattern.
I think I can use `handle()` here and https://github.com/line/armeria/pull/3584/files#diff-bdfe29918f02ed24a4fd7e4ce1a0dc710ec18ee74f61d7d518533613d27f7b06R228